extras/tsl2561/tsl2561.h

```diff
@@ -0,0 +1,125 @@
+/*
+ * Part of esp-open-rtos
+ * Copyright (C) 2016 Brian Schwind (https://github.com/bschwind)
+ * BSD Licensed as described in the file LICENSE
+ */
+
+#ifndef __TSL2561_H__
+#define __TSL2561_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+// I2C Addresses
+#define TSL2561_I2C_ADDR_VCC 0x49
```

Comment:
I moved most of the defines into the .c file, but there are a few I left in the header which you can use when calling particular TSL2561 functions, such as when choosing an I2C address or an integration time constant.